lib/client.js

```diff
@@ -82,6 +87,69 @@ Client.prototype.listArchives = function () {
 Client.prototype.deleteArchive = function () {
 };
 
+Client.prototype.setArchiveLayout = function setArchiveLayout(opts, cb) {
```

Comment:
What if we created a re-usable method called `setLayout` which can be used for both `broadcast` and `archiving`?

Reply:
I prefer having the methods of the Client object correspond one-to-one with the REST API methods. (There are separate REST methods for setting an archive layout and setting a broadcast layout.)